versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.12.0
+DATE TBD
+
+### Improvements
+- Use `ingressClassName` field for ingresses instead of the deprecated `kubernetes.io/ingress.class` annotation
+- Okteto daemonset no longer mounts host volumes nor runs in privilege mode
+- Improve pipeline jobs lifecycle and properly handle timeouts. Logs for pipelines that time out are no longer lost
+- Invalid ingresses not accepted by the nginx ingress controller are now caught by the webhook
```

Comment:
is this a breakign change?  if so, we should add a section explicitly calling those out.